src/Octoshift/Handlers/WaitForMigrationCommandHandler.cs

```diff
@@ -108,20 +108,22 @@ private async Task WaitForRepositoryMigration(string migrationId, string githubP
             if (RepositoryMigrationStatus.IsSucceeded(state))
             {
                 _log.LogSuccess($"Migration {migrationId} succeeded for {repositoryName}");
+                _log.LogInformation($"Migration log available at {url} or by running `gh gei download-logs`");
```

Comment:
```suggestion
                _log.LogInformation($"Migration log available at {migrationLogUrl} or by running `gh gei download-logs`");
```

Reply:
Resolved in b8273b92baf683a9429245d93d66d735d24127a4 :tada: